prow/plugins/verify-owners/verify-owners.go

```diff
@@ -138,50 +137,51 @@ func handle(ghc githubClient, gc *git.Client, log *logrus.Entry, pre *github.Pul
 		}
 		// Check labels against blacklist
 		if sets.NewString(labels...).HasAny(labelsBlackList...) {
-			wrongOwnersFiles[f] = fmt.Errorf("OWNERS file contains blacklisted labels: %s", sets.NewString(simple.Config.Labels...).Intersection(sets.NewString(labelsBlackList...)).List())
+			wrongOwnersFiles[f] = fmt.Sprintf("File contains blacklisted labels: %s.", sets.NewString(labels...).Intersection(sets.NewString(labelsBlackList...)).List())
 			continue
 		}
 		// Check approvers isn't empty
 		if filepath.Dir(f) == "." && len(approvers) == 0 {
-			wrongOwnersFiles[f] = errors.New("no approvers in the root directory OWNERS file")
+			wrongOwnersFiles[f] = "No approvers defined in file."
 			continue
 		}
 	}
 	// React if we saw something.
 	if len(wrongOwnersFiles) > 0 {
+		s := "s"
+		if len(wrongOwnersFiles) == 1 {
+			s = ""
+		}
 		if err := ghc.AddLabel(org, repo, pre.Number, invalidOwnersLabel); err != nil {
 			return err
 		}
-		log.Debugf("Creating a review for %d OWNERS files.", len(wrongOwnersFiles))
+		log.Debugf("Creating a review for %d %s file%s.", len(wrongOwnersFiles), ownersFileName, s)
 		var comments []github.DraftReviewComment
 		for errFile, errMsg := range wrongOwnersFiles {
-			resp := fmt.Sprintf("Invalid OWNERS file: %v", errMsg)
+			resp := errMsg
 			comments = append(comments, github.DraftReviewComment{
-				Path: errFile,
-				Body: resp,
+				Path:     errFile,
+				Body:     resp,
+				Position: 1,
 			})
 		}
 		// Make the review body.
-		s := "s"
-		if len(wrongOwnersFiles) == 1 {
-			s = ""
-		}
-		response := fmt.Sprintf("%d invalid OWNERS file%s", len(wrongOwnersFiles), s)
+		response := fmt.Sprintf("%d invalid %s file%s", len(wrongOwnersFiles), ownersFileName, s)
 		err := ghc.CreateReview(org, repo, pre.Number, github.DraftReview{
 			Body:     plugins.FormatResponseRaw(pre.PullRequest.Body, pre.PullRequest.HTMLURL, pre.PullRequest.User.Login, response),
 			Action:   github.Comment,
 			Comments: comments,
```

Comment:
We should also specify the `commit_id` field to ensure that the review is posted for the right commits even if the PR is updated while we are posting the review. https://developer.github.com/v3/pulls/reviews/#input